reconciletags/config_integrity_test.py

```diff
@@ -0,0 +1,51 @@
+"""Tests to check the integrity of json config files.
+
+These tests assume that the json configs live in a top-level
+folder named config."""
+
+import glob
+import json
+import subprocess
+import unittest
+
+
+class ReconcilePresubmitTest(unittest.TestCase):
+
+    def _get_digests(self, repo):
+        output = json.loads(
```

Comment:
probably want to do this in a try/except block, so if the call fails the client doesn't spit out a nasty error. instead you can just tell them to install gcloud.